examples/demo.php

```diff
@@ -1,16 +1,14 @@
 <?php
 
-use Clue\React\Ssdp\Client;
-
 require __DIR__ . '/../vendor/autoload.php';
 
-$client = new Client();
+$client = new Clue\React\Ssdp\Client();
 
 $client->search()->then(
     function () {
         echo 'Search completed' . PHP_EOL;
     },
-    function($e) {
+    function(Excepetion $e) {
```

Comment:
Typo:

```suggestion
    function(Exception $e) {
```

Reply:
Thanks @clue, should be fixed right now. 👍